packages/api/helper/googleSheetsAPIConnection.js

```diff
@@ -0,0 +1,31 @@
+const { google } = require('googleapis');
+
+const client = new google.auth.JWT(
+  process.env.GOOGLE_SHEETS_CLIENT_EMAIL,
+  null,
```

Comment:
Could you add a comment here to indicate what this `null` represents, to aid in legibility?

Reply:
null is a place holder. JWT accepts a parameter of key file which we do not use, instead we used env vars , however it will not run properly with out this in place.